### ISSUE TITLE
Verbose output of pkg_install

### ISSUE THREAD

**Issue by gongyh** [r-lib/pak#517]
When using pak in Dockerfile, some packages need long time to compile (such as httpuv, sass, etc.), so online build services (such as quay, dockerhub) receive no output for a long time (hangs and fail). Could it be feasible to add a verbose parameter to output some debug information?

**Comment by gaborcsardi** [comment]
Yeah, I think we could produce some summary output about what is going on, every minute or so?

**Comment by gongyh** [comment]
every minute is fine. look forwarding to this feature.
